### ISSUE TITLE
Oppiabot should tag the assignes in the issues if it is a blocking bug. 

### ISSUE THREAD

**Issue by SAEb-ai** [oppia/oppiabot#226]
**For blocking bug issues:**

1. If some contributor is assigned :
     - Oppiabot should ping the assigned contributor if no pull request is linked to the issue after
       24 hours.
     - Oppiabot should ping the release team if no pull request is linked to the issue after 24 hours.

2. If no contributor is assigned :
    - Oppiabot should ping the release team about the issue after 24 hours of issue creation to assign somebody.

@vojtechjelinek @jameesjohn I think if this could be achieved by oppiabot, then the blocking bugs could be tracked and solved more faster. Does it make sense? PTAL.

**Comment by vojtechjelinek** [comment]
Hmm, sounds interesting but I think that the blocking bugs need a bit more careful and individual handling, also it is the responsibility of the release coordinator to do this and they should have a good overview of what is happening with the blocking bugs. For personal experience most blocking bugs need a discussion with the person that should be fixing them and usually there is a lot of stuff going on behind that would be to complicated to implement in Oppiabot.

**Comment by jameesjohn** [comment]
I tend to lean towards @vojtechjelinek.
I think the feature would be nice to have, but if the bug is taking some time to get fixed, it might seem as if the bot is spamming if it continually pings the release team every 24 hours.

**Comment by vojtechjelinek** [comment]
Closing for now.

**Comment by SAEb-ai** [comment]
> I tend to lean towards @vojtechjelinek.
> I think the feature would be nice to have, but if the bug is taking some time to get fixed, it might seem as if the bot is spamming if it continually pings the release team every 24 hours.

@jameesjohn Couldn't we just sustain the bot to inform the release coordinator only once in order to avoid spamming. 
@vojtechjelinek @jameesjohn  I totally agreee to your point. But I think that blocking bugs being the top priorities before the release, shouldn't you think that the bot should do some work there too?

**Comment by vojtechjelinek** [comment]
I think that there was no request from the releases for this, we have many other issues for Oppiabot. This can be reconsidered if there is a request from releasers for this.
